index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <h2>Welcome</h2>
+        <nav class="navbar">
+            <img src="/assets/Union.png" alt="">
+        </nav>
+    </header>
+    <main>
+      <section class="intro_section">
+        <h1> Hey there. <br> I'm Favour <br>
+            <span>I'm a software developer</span>
+        </h1>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </section>
+        <ul>
+            <li>
+                <a href="www.facebook.com"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                <a href="twitter.com/Fabimworld2536"><i class="fa fa-twitter" aria-hidden="true"></i></a>
```

Comment:
Please, as recommended by the previous reviewer, try to follow the design as closely as possible. You did great at improving it from the previous.

Not only does it look better, but you'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible. So, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit.

Follow the original vision if it's pretty and works, it will be a wonderful website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185405662-c303e09e-cf50-45ab-be60-0070242c241f.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185404069-02201685-b382-44dc-80ec-f3ed1d892f1d.png)